specs/digital_root_spec.rb

```diff
@@ -1,4 +1,9 @@
 describe "#digital_root" do
+  before(:each) do
+    expect_any_instance_of(Integer).to_not receive(:to_s)
```

Comment:
Let's add a blacklist spec for the `Integer#digits` method as well. And add the following note to the problem description file:

" \*\*Do NOT use the built-in `Integer#to_s`, or `Integer#digits` methods in your implementation.\*\*"